src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,14 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={user.email}>
```

Comment:
if you click on this you'll se an error 
![image](https://user-images.githubusercontent.com/122989585/219612445-330e761b-2a3a-49d2-accf-a1a96bfc35bb.png)
Check how to write correct mail links here pls: 
https://css-tricks.com/snippets/html/mailto-links/